src/test/debuginfo/msvc-pretty-enums.rs

```diff
@@ -8,61 +7,61 @@
 //       so the best we can do is to make sure we are generating the right debuginfo
 
 // cdb-command: dx -r2 a,!
-// cdb-check:a,!              [Type: enum$<core::option::Option<enum$<msvc_pretty_enums::CStyleEnum>>, 2, 16, Some>]
-// cdb-check:    [+0x000] dataful_variant  [Type: enum$<core::option::Option<enum$<msvc_pretty_enums::CStyleEnum>>, 2, 16, Some>::Some]
+// cdb-check:a,!              : Some({...}) [Type: enum$<core::option::Option<enum$<msvc_pretty_enums::CStyleEnum> >, 2, 16, Some>]
```

Comment:
The `: Some({...})` part is also not printed on CI:
https://github.com/rust-lang-ci/rust/runs/2962304803#step:25:10518

I think that is what the comment above is referring to, right @wesleywiser?

Reply:
Looking at the output in the build, the same discrepancy applies to the other cases in this file too, it seems.

Reply:
> I think that is what the comment above is referring to, right @wesleywiser?

Yeah, that's correct.